claims_to_nodes/pipe.py

```diff
@@ -92,14 +92,14 @@ def process_claim(raw_claim):
                 source_node = get_or_create_node(raw_claim['sourceURI'], raw_claim)
 
             # Create the claim node
-            claim_uri = raw_claim['claimAddress'] or 'https://live.linkedtrust.us/claims/{}'.format(raw_claim['id'])
+            claim_uri = raw_claim['claimAddress'] or make_subject_uri(raw_claim['id'])
```

Comment:
if we're going to do this we could also include the logic about claimAddress, and just do  `make_subject_uri(raw_claim)`  :-)

Reply:
yeah, good point :"